fastlane/Fastfile

```diff
@@ -233,24 +236,25 @@ platform :ios do
   #####################################################################################
   desc 'Trigger the final release build on CI'
   lane :finalize_release do |options|
+    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+
     ios_finalize_prechecks(options)
-    unless ios_current_branch_is_hotfix
-      UI.message('Checking app strings translation status...')
-      check_translation_progress(
-        glotpress_url: 'https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/',
-        abort_on_violations: false
-      )
-
-      UI.message("Checking release notes strings translation status...")
-      check_translation_progress(
-        glotpress_url: 'https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/release-notes/',
-        abort_on_violations: false
-      )
-
-      ios_update_metadata(options)
-      ios_lint_localizations(input_dir: 'WooCommerce/Resources', allow_retry: true)
-      ios_bump_version_beta
-    end
+
+    UI.message('Checking app strings translation status...')
+    check_translation_progress(
+      glotpress_url: 'https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/',
+      abort_on_violations: false
+    )
+
+    UI.message("Checking release notes strings translation status...")
+    check_translation_progress(
+      glotpress_url: 'https://translate.wordpress.com/projects/woocommerce/woocommerce-ios/release-notes/',
+      abort_on_violations: false
+    )
+
+    ios_update_metadata(options)
+    ios_lint_localizations(input_dir: 'WooCommerce/Resources', allow_retry: true)
```

Comment:
This PR copied the code from the previous version, which I think is fine.

I wonder if it would be worth following up with using `File.join(PROJECT_ROOT, 'WooCommerce', 'Resources')` (where `PROJECT_ROOT` is the `project_root` defined above, but extracted as a constant).

Reply:
See [comment above](https://github.com/woocommerce/woocommerce-ios/pull/4703#discussion_r680886239).